packages/workbox-core/_private/cacheWrapper.mjs

```diff
@@ -36,6 +36,16 @@ import '../_version.mjs';
  * @memberof module:workbox-core
  */
 const putWrapper = async (cacheName, request, response, plugins = []) => {
+  if (!response) {
+    if (process.env.NODE_ENV !== 'production') {
+      logger.error(`Request '${getFriendlyURL(request.url)}' had no URL.`,
```

Comment:
`had no URL` doesn't seem like the right message to log.

How about `"Can't add a non-existent response to the cache for ${getFriendlyURL(request.url)}."`, or something like that?